api/test/run_test.sh

```diff
@@ -19,7 +19,7 @@ if [[ $succeeded -eq 0 ]]; then
 	printf "${green}:-) All tests passed${endcol}\n"
 else
     printf "Full system logs:\n"
-    docker-compose logs
+    # docker-compose logs
```

Comment:
I really think we should just delete it and re-prioritize the issue to make it work in CI. Or else I'm happy to add an `if CI` in here.

This line literally makes local test debugging impossible and I actually know what is happening. For a new contributor it would be baffling.

Reply:
I'm fine with that, honestly. It seems like it's been a real pain for API tests!